### PR TITLE
Update file-with-help copy and add deadline notice #177748874

### PR DESCRIPTION
**Pull request by coltborg**
[Add notice to file-with-help #177748874](https://www.pivotaltracker.com/story/show/177748874)

# What this PR does

- Adds a notice about the tax deadline extension to the `questions/file-with-help` page
- Aligns the `questions/file-with-help` content to the new designs
- Have both English and Spanish copy
- Update any tests that reference the previous copy for this page

### CODE REVIEW

**Comment by embarnard** [comment]
Oh wow thank you for this overview! This is super helpful for the review

File: app/views/questions/file_with_help/edit.html.erb [hunk 1]
```diff
@@ -10,24 +10,18 @@
...
+      <p><%= t("views.questions.file_with_help.extension_notice_html") %></p>
+    </div>
 
     <%= link_to next_path, class: "button button--primary text--centered button--wide spacing-above-60" do %>
       <%=t("general.continue") %>
     <% end %>
+
+    <%= link_to t("views.questions.file_with_help.diy"), diy_file_yourself_path %>
+
+    <!-- TODO: run i18n health and delete any unused -->
```
Comment (embarnard) [inline app/views/questions/file_with_help/edit.html.erb:25]: Should we remove this comment?

**Review by embarnard — APPROVED** [review]
Looks good! 😄  Just the one note to remove the "todo" comment

File: app/views/questions/file_with_help/edit.html.erb (hunk 1 above)
Comment (coltborg) [reply app/views/questions/file_with_help/edit.html.erb]: Yes! Removed.

**Pull request merged**